src/System.Diagnostics.EventLog/tests/EventLogReaderTests/EventLogConfigurationTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Eventing.Reader;
+using System.Threading;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    public class EventLogConfigurationTests
+    {
+        [ConditionalFact(typeof(Helpers), nameof(Helpers.SupportsEventLogs))]
+        public void Properties_DoNotThrow()
+        {
+            var configuration = new EventLogConfiguration("Application");
+
+            var LogName = configuration.LogName;
```

Comment:
please use the exact type on LHS for all the properties here and use name with lowercase